solution/ui/regulations/eregs-component-lib/jsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
```

Comment:
These .jsconfig files have been added for better LSP performance -- they allow editor LSP integrations know where the aliases reference